src/main/java/step5/domain/Cars.java

```diff
@@ -0,0 +1,52 @@
+package step5.domain;
+
+import step5.MovingStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.OptionalInt;
+import java.util.stream.Collectors;
+
+public class Cars {
+	private final List<Car> racingCars;
+
+	public Cars(List<Car> racingCars) {
+		this.racingCars = racingCars;
+	}
+
+	public Cars(String names) {
+		racingCars = new ArrayList<>();
+		String[] separatedNames = names.split(",");
+		for (String separatedName : separatedNames) {
+			racingCars.add(new Car(separatedName));
+		}
+	}
+
+	public List<Car> getRacingCars() {
+		return racingCars;
+	}
+
+	public Cars findWinner() {
+		return new Cars(
+				racingCars.stream()
+						.filter(car -> car.getRecord() == maxRecord())
+						.collect(Collectors.toList())
+		);
+	}
```

Comment:
우승자를 구할때 최대 거리는 한번만 구해도 되지 않을까 하는데 어떻게 생각하시나요 :) ?
```suggestion
	public Cars findWinner() {
		int maxRecord = getMaxRecord();
		return new Cars(
				racingCars.stream()
						.filter(car -> car.getRecord() == maxRecord)
						.collect(Collectors.toList())
		);
	}
```